client/src/app/project-registration/project-id/project-id.component.ts

```diff
@@ -197,6 +224,15 @@ export class ProjectIdComponent implements OnInit {
     } else {
       this.otherTechnology = '';
     }
+  }
+
+  private onOrganChange(val: any) {
+    const organ = this.metadataFormService.cleanFormData(val);
+    if (organ && organ.length > 0) {
+      this.organ = ProjectIdComponent.sanitiseOrgan(organ[0]['ontology_label']);
+    } else {
+      this.organ = '';
+    }
     this.generateProjectId();
```

Comment:
Why did you move the generateProjectId() to onOrganChange()? Is the organ value being used in the project id?

Reply:
i think the project id should still generated every time technology value changes.

Reply:
This was a mistake, thanks. Fixed now